.github/workflows/sync-metadata.yaml

```diff
@@ -49,4 +50,13 @@ jobs:
             draft: false
             signoff: true
             delete-branch: true
-            token: ${{ secrets.CI_TOKEN }}
\ No newline at end of file
+            token: ${{ secrets.CI_TOKEN }}
+
+        - name: PR Creation Summary
+          run: |
+            if [ "${{ steps.create-pr.outputs.pull-request-number }}" != "" ]; then
+              echo "✅ Pull request created: ${{ steps.create-pr.outputs.pull-request-url }}"
+              echo "PR #${{ steps.create-pr.outputs.pull-request-number }}"
+            else
+              echo "ℹ️ No metadata changes detected - no pull request created"
+            fi
```

Comment:
It took me some time to trace through exactly what this code path was doing. I'd added a bit of messaging to hopefully make this a bit clearer.